web/sites/default/settings.env.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+// First, try to determine where we are running.
+$infra = match(true) {
+  getenv('IS_DDEV_PROJECT') == 'true' => 'ddev',
```

Comment:
@hussainweb can we make this conditional based on the hosting provider and include the code for all hosting providers? I understand we might not be able to test it but that can be done in projects.